html5lib/tests/tree_construction.py

```diff
@@ -51,14 +51,17 @@ def runtest(self):
         fragmentContainer = self.test['document-fragment']
         expected = self.test['document']
         expectedErrors = self.test['errors'].split("\n") if self.test['errors'] else []
+        script = True
+        if 'script-off' in self.test:
+            script = False
```

Comment:
I wonder if this is a horrible suggestion, but I wonder if it's worthwhile doing something like `if 'script-on' in self.test and '<script' in self.test['data']: pytest.skip("scripting test")`?